integrated_channels/canvas/client.py

```diff
@@ -309,7 +314,18 @@ def cleanup_duplicate_assignment_records(self, courses):
             # Continue iterating over assignment responses while more paginated results exist or until the page count
             # limit is hit
             while more_pages_present and current_page_count < 150:
+                start_time = time.time()
                 resp = self.session.get(canvas_assignments_url)
+                duration_seconds = time.time() - start_time
+                self.IntegratedChannelAPIRequestLogs.store_api_call(
+                    enterprise_customer=self.enterprise_configuration.enterprise_customer,
+                    enterprise_customer_configuration_id=self.enterprise_configuration.id,
+                    endpoint=canvas_assignments_url,
+                    payload='',
+                    time_taken=duration_seconds,
+                    status_code=resp.status_code,
+                    response_body=resp.text,
+                )
```

Comment:
Question: Shouldn't we be using `stringify_and_store_api_record` here just like we are doing at all other places?